src/sleep/meson.build

```diff
@@ -1,9 +1,49 @@
 # SPDX-License-Identifier: LGPL-2.1-or-later
 
+libsleep_core_sources = files(
+        'hibernate-util.c',
+        'sleep-config.c',
+)
+
+libsleep_core = static_library(
```

Comment:
An entire new library just for a few helpers seems overkill to me. It all adds up to the setup and build time. Why not simply leave it in shared?